common/lib/capa/capa/responsetypes.py

```diff
@@ -709,6 +709,12 @@ def setup_response(self):
         self.correct_choices = set([choice.get(
             'name') for choice in correct_xml])
 
+        incorrect_xml = self.xml.xpath('//*[@id=$id]//choice[@correct="false"]',
+                                     id=self.xml.get('id'))
+
+        self.incorrect_choices = set([choice.get(
+            'name') for choice in incorrect_xml])
```

Comment:
It's broken in a way identical to line 706 (correct_xml = ...), since I copied it from there. I can fix that one too if you like.